python-checks/src/test/resources/checks/datetime_constructor_use_timezone_aware.py

```diff
@@ -0,0 +1,40 @@
+def some_function():
+    from datetime import datetime
+    datetime.utcnow() # Noncompliant {{Using timezone aware "datetime"s should be preferred over using "datetime.datetime.utcnow" and "datetime.datetime.utcfromtimestamp"}}
+   #^^^^^^^^^^^^^^^^^
+    timestamp = 1571595618.0
+    datetime.utcfromtimestamp(timestamp) # Noncompliant {{Using timezone aware "datetime"s should be preferred over using "datetime.datetime.utcnow" and "datetime.datetime.utcfromtimestamp"}}
+   #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+def other_function():
+    import datetime
+    timestamp = 1571595618.0
+    datetime.datetime.utcnow() # Noncompliant
+    datetime.datetime.utcfromtimestamp(timestamp) # Noncompliant
+
+    from datetime import datetime as dt
+    dt.utcnow() # Noncompliant
+    dt.utcfromtimestamp(timestamp) # Noncompliant
+
+def unknown_symbols():
+
+    unrelated_call()
+
+    datetime.datetime.utcnow() # FN because there is no symbol
+    datetime.datetime.utcfromtimestamp() # FN because there is no symbol
```

Comment:
If the symbols are unknown because not imported, I think it's fair to not report any issue: they could be anything at this point. So it's not FN (FN kind of implies we'd like to raise an issue here - which we don't).